src/components/Snippet.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+import SnippetFooter from "./SnippetFooter";
+import SnippetHeader from "./SnippetHeader";
```

Comment:
UI로 보여지는 위치에 따라 순서대로 import 해 주시면 좋을 것 같습니다!😊

```suggestion
import SnippetHeader from "./SnippetHeader";
import SnippetFooter from "./SnippetFooter";
```